src/rgw/rgw_common.h

```diff
@@ -917,6 +924,8 @@ struct RGWBucketInfo {
 
   std::map<std::string, uint32_t> mdsearch_config;
 
+  rgw::cksum::Type cksum_type = rgw::cksum::Type::none;
```

Comment:
this is an artifact from my original PR which stored a checksum preference with the bucket info;  Maybe this should go away...